content/_index.md

```diff
@@ -28,6 +28,12 @@ These resources may be helpful in preparing for enforcement discussions:
 - [jQuery Foundation Code of Conduct Enforcement Manual](https://js.foundation/community/code-of-conduct/enforcement)
 - [How Mozilla is Making Code of Conduct Enforcement Real - and Scaling It](https://medium.com/mozilla-open-innovation/how-were-making-code-of-conduct-enforcement-real-and-scaling-it-3e382cf94415)
 
+It's equally as important to consider the opportunity of this document as a driver of positive change for discussion, and action that are critical for building safe and inclusive communities. Some resources that might be helpful in leveraging this document for inclusive design:
```

Comment:
"to consider this document…" and "for discussion and actions…"

What do you think?

Reply:
Yes, less wordy, more clear!

Reply:
OK, let me know when you've pushed the revision and I'll happily merge.

Reply:
Also, if you can DM me on twitter (@CoralineAda) I have something to share with you. Sorry, I know I follow you but I can't remember your handle!